lib/LOLIN_EPD/src/LOLIN_IL3897.cpp

```diff
@@ -220,6 +220,16 @@ LOLIN_IL3897::LOLIN_IL3897(int width, int height, int8_t DC, int8_t RST, int8_t
     red_bufsize = bw_bufsize;
 }
 
+/**************************************************************************/
+/*!
+    @brief default destructor
+*/
+/**************************************************************************/
+LOLIN_IL3897::~LOLIN_IL3897() {
+  free(bw_buf);
+  free(red_buf);
```

Comment:
It inherits from `LOLIN_EPD` so delete it in the base class.
Then it is freed from any derived class which is destructed.

Reply:
There it is also freed. Would that mean the empty destructors aren't needed at all, just the `virtual` declaration on the base class destructor?

Reply:
I had hopes that freeing the memory would release more memory when disabling the plugin, as it is claiming quite a lot when started (~7kB on ESP8266), but releases only about 2k on disable 🤔. Haven't yet found where all that memory is allocated.

Reply:
Found an answer, removed the destructors, just leave it `virtual` in the base class.